main.go

```diff
@@ -11,75 +11,148 @@
 package main
 
 import (
+	"bufio"
 	"bytes"
 	"fmt"
 	"github.com/frankbille/go-wxr-import"
 	"github.com/writeas/godown"
+	"github.com/writeas/nerds/store"
 	"go.code.as/writeas.v2"
 	"io/ioutil"
 	"log"
 	"os"
 	"regexp"
+	"strings"
 )
 
 var (
 	commentReg = regexp.MustCompile("(?m)<!-- ([^m ]|m[^o ]|mo[^r ]|mor[^e ])+ -->\n?")
 )
 
-func main() {
-	if len(os.Args) < 2 {
-		//errQuit("usage: wp-import https://write.as filename.xml")
-		errQuit("usage: wp-import filename.xml")
+// Print the usage spec to the terminal and exit cleanly
+func printUsage(help bool) {
+	usage := "usage: wp-import [-h|--help] [-i instance] [-f] filename.xml"
```

Comment:
As a bonus of using the `flag` package (as mentioned in another comment), we'll get all this output for free. So we probably won't need this func either.